tests/Microsoft.Fx.Portability.MetadataReader.Tests/ReflectionMetadataInfoTests.cs

```diff
@@ -76,6 +76,8 @@ public FilePathAssemblyFile(string path)
             public string Version => FileVersionInfo.GetVersionInfo(_path).FileVersion;
 
             public Stream OpenRead() => File.OpenRead(_path);
+
+            public bool SkipBinaryIfPackageExists { get { return false; } }
```

Comment:
Please stay consistent with the rest of the file; `SkipBinaryIfPackagesExist => false;`

Reply:
Done